packages/browser/playground/nextjs/package.json

```
@@ -5,13 +5,13 @@
     "packageManager": "pnpm@9.15.4",
     "scripts": {
         "clean-react": "cd ../../react && pnpm run clean-node-modules",
-        "dev": "next dev",
+        "dev": "pnpm run clean-react && next dev",
         "dev-crossdomain": "NEXT_PUBLIC_CROSSDOMAIN=1 next dev --experimental-https",
-        "install-deps": "pnpm install && pnpm run build-posthog-js && pnpm install --ignore-workspace",
+        "install-deps": "pnpm run build-posthog-js && pnpm install",
```

vercel deployment silently fails with this config, from memory it's because of multiple react installation